qiita_db/analysis.py

```diff
@@ -883,47 +895,44 @@ def _build_mapping_file(self, samples):
                      WHERE filepath_type = 'qiime_map'
                         AND artifact_id IN (SELECT *
                                             FROM qiita.find_artifact_roots(%s))
-                     ORDER BY filepath_id DESC"""
-            _id, fp = qdb.util.get_mountpoint('templates')[0]
+                     ORDER BY filepath_id DESC LIMIT 1"""
+            _, fp = qdb.util.get_mountpoint('templates')[0]
```

Comment:
And this is the same that `fp = qdb.util.get_mountpoint('templates')[0][0]`. Probably adding a comment saying that we are only interested in the filepath but not in the id.